src/replxx_impl.cxx

```diff
@@ -809,10 +809,23 @@ void Replxx::ReplxxImpl::render( HINT_ACTION hintAction_ ) {
 		IOModeGuard ioModeGuard( _terminal );
 		_highlighterCallback( _utf8Buffer.get(), colors );
 	}
+	std::unordered_map<Color, Color> brighter_colormap{
```

Comment:
Maybe it is better to use some kind of callback here instead of copy-pasting more or less the same code in the library and in the ClickHouse?